src/modules/booking/components/bookings-list/bookings-list.tsx

```diff
@@ -64,9 +72,53 @@ export const BookingsList: FC<BookingsListProps> = () => {
           (group) =>
             !group.hidden && (
               <div className={styles.group} key={group.id}>
-                <Typography className={styles.title} variant='body1'>
-                  {group.title}
-                </Typography>
+                <div className='flex items-center justify-between'>
+                  <Typography className={styles.title} variant='body1'>
+                    {group.title}
+                  </Typography>
+                  {group.id === 'upcomming' && (
+                    <DropdownMenu
```

Comment:
You can create a separate component `MyBookingActions` for this dropdown. And reuse it on the 'list' and 'calendar' tabs.

```tsx
export const MyBookingActions: FC = () => {
  return (
    <DropdownMenu... />
  )
}
```

Additionally, on desktop, it should be only `Create` button (the same as on desktop calendar).